ghost/i18n/locales/pt/portal.json

```diff
@@ -1,16 +1,16 @@
 {
-    "(save {{highestYearlyDiscount}}%)": "",
+    "(save {{highestYearlyDiscount}}%)": "(Poupa {{highestYearlyDiscount}}%)",
     "{{amount}} days free": "{{amount}} dias gratuitos",
     "{{amount}} off": "Desconto de {{amount}}",
     "{{amount}} off for first {{number}} months.": "Desconto de {{amount}} nos primeiros {{number}} meses.",
     "{{amount}} off for first {{period}}.": "Desconto de {{amount}} durante o primeiro {{period}}.",
     "{{amount}} off forever.": "Desconto de {{amount}} para sempre.",
     "{{discount}}% discount": "{{discount}}% de desconto",
-    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} deixará de receber a newsletter {{newsletterName}}.",
-    "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} deixará de receber emails quando alguém responder aos seus comentários.",
-    "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} deixará de receber esta newsletter.",
+    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} deixarás de receber a newsletter {{newsletterName}}.",
+    "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} deixarás de receber emails quando alguém responder aos teus comentários.",
+    "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} deixarás de receber esta newsletter.",
     "{{trialDays}} days free": "{{trialDays}} dias grátis",
-    "+1 (123) 456-7890": "",
+    "+1 (123) 456-7890": "+351 912 345 678",
```

Comment:
This is a placeholder number, we don't need to change it :) You can leave the empty string